source/java/index.rst

```diff
@@ -0,0 +1,34 @@
+Java 运行环境
+=============
+
+:教程作者: `田冬冬 <https://me.seisman.info/>`__ (作者)、`姚家园 <https://github.com/core-man>`__ (审稿)
+:最近更新时间: 2020-12-31
+
+运行 Java 语言写的程序时，需要安装 Java 运行环境。
```

Comment:
Do we want to explicitly indicate which software uses java? So we know we have to revise the content in that software when we change something here.

Reply:
> revise the content in that software when we change something here.

I don't think we need to make any changes as they are totally independent, right?

Reply:
I mean it's better to know which software uses java in the java tutorial. If we make some changes in java in the future which potentially may lead us to revise the java reference in that software, we may know which software rst file needs to be revised.

Reply:
In a separate PR, I will revise the file taup/install.rst, delete the long paragraph about installing java, and add a link to this file. In the future, if we really want to know which software depends on java, we only need to run a simple `grep` command.

Reply:
Okay~